.github/workflows/holochain-build-and-test.yml

```diff
@@ -8,9 +8,12 @@ on:
         required: false
         type: string
         default: "."
+    secrets:
+      TS_CACHIX_TOKEN:
+        required: true
```

Comment:
This needs tidying up, we won't need a new token because both caches can use the same token